src/DynamoCoreWpf/Views/CodeCompletion/CodeCompletionEditor.xaml.cs

```diff
@@ -52,6 +61,50 @@ public CodeCompletionEditor(NodeViewModel nodeViewModel)
         }
 
         /// <summary>
+        /// Set focus to the editor.
+        /// </summary>
+        public void SetFocus()
+        {
+            InnerTextEditor.Focus();
+        }
+
+        #region Dependency Property
+        /// <summary>
+        /// Dependency property.
+        /// </summary>
+        public static readonly DependencyProperty CodeProperty =
+            DependencyProperty.Register(
+                "Code",
+                typeof(string),
+                typeof(CodeCompletionEditor),
+                new PropertyMetadata((obj, args) =>
+                {
+                    var target = (CodeCompletionEditor)obj;
+                    target.Code = (string)args.NewValue;
+                })
+            );
+        #endregion
+
+        #region Properties
+
+        /// <summary>
+        /// Set the content of the editor.
+        /// </summary>
+        public string Code
+        {
+            get
+            {
+                throw new NotImplementedException();
```

Comment:
I was wondering about this myself (I think I wrote the original code), perhaps we should put the original comments back for clarity:

```csharp
// Since this property a one way binding from source (CodeBlockNodeModel)
// to target (this class), the getter should never be called.
```